Source/WebKit/NetworkProcess/NetworkDataTaskBlob.cpp

```diff
@@ -278,7 +278,7 @@ void NetworkDataTaskBlob::dispatchDidReceiveResponse()
     // as if the response had a Content-Disposition header with the filename parameter set to the File's name attribute.
     // Notably, this will affect a name suggested in "File Save As".
 
-    didReceiveResponse(WTFMove(response), NegotiatedLegacyTLS::No, PrivateRelayed::No, std::nullopt, [this, protectedThis = WTFMove(protectedThis)](PolicyAction policyAction) {
+    didReceiveResponse(WTFMove(response), NegotiatedLegacyTLS::No, PrivateRelayed::No, String { }, std::nullopt, [this, protectedThis = WTFMove(protectedThis)](PolicyAction policyAction) {
```

Comment:
I am thinking if I should just mark this value as "default-for-Blob" instead of an empty string.